utils.sh

```diff
@@ -26,10 +26,15 @@ function create_cluster() {
     $GOPATH/src/github.com/openshift-metalkube/kni-installer/bin/kni-install --dir "${assets_dir}" --log-level=debug create manifests
 
     # TODO - consider adding NTP server config to install-config.yaml instead
-    if host clock.redhat.com ; then
-        cp assets/templates/99_worker-chronyd-redhat.yaml.optional assets/templates/99_worker-chronyd-redhat.yaml
-        cp assets/templates/99_master-chronyd-redhat.yaml.optional assets/templates/99_master-chronyd-redhat.yaml
+    host clock.redhat.com > /dev/null && NTP_SERVERS="clock.redhat.com"
```

Comment:
this means that if run within redhat network, you ll get this specific ntp server regarless of the option set for NT_SERVERS

Reply:
Fixed I believe